xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -203,6 +226,12 @@ struct AppModel: ModelProtocol {
         environment: AppEnvironment
     ) -> Update<AppModel> {
         switch action {
+        case .recoveryPhrase(let action):
```

Comment:
Handle boxed child actions by passing down to child component update function using update function synthesized by CursorProtocol.